source/AMDSClientUi.cpp

```diff
@@ -334,6 +335,21 @@ void AMDSClientUi::resetActiveContinuousConnection(QString serverIdentifier)
 {
 	QStringList activeContinuousClientRequestKeys = clientAppController_->getActiveSocketKeysByServer(serverIdentifier);
 
-	activeContinuousConnectionComboBox_->clear();
-	activeContinuousConnectionComboBox_->addItems(activeContinuousClientRequestKeys);
+	bool updateActiveContiuousConnectionComboBox = false;
+	if (activeContinuousClientRequestKeys.length() == activeContinuousConnectionComboBox_->count()) {
+		for (int i = 0; i < activeContinuousConnectionComboBox_->count(); i++) {
+			QString itemText = activeContinuousConnectionComboBox_->itemText(i);
+			if (!activeContinuousClientRequestKeys.contains(itemText)) {
+				updateActiveContiuousConnectionComboBox = true;
+				break;
```

Comment:
Rather than using a break statement, you could put the flag inside the for-loop comparison clause.
```
for (int i = 0, size = activeContinuousConnectionComboBox_->count(); i < size && !updateActiveContinuousConnectionComboBox; i++)
     ...
```